doc/version-history.rst

```diff
@@ -0,0 +1,14 @@
+.. _Version_History:
+
+===============
+Version History
+===============
+
+Unreleased
+==========
```

Comment:
If the package is usable in this form then I encourage you to release this version, which requires assigning a version here. If the API or code is still in flux then I suggest starting with `0.1.0`, and waiting for `1.0.0` until things stabilize.

Reply:
Ping

Reply:
I'll release when a more tickets are resolved.